streams/src/main/java/org/apache/kafka/streams/processor/To.java

```diff
@@ -89,4 +89,11 @@ public int hashCode() {
         throw new UnsupportedOperationException("To is unsafe for use in Hash collections");
     }
 
+    @Override
+    public String toString() {
+        return "To{" +
+               "childName='" + childName + '\'' +
+               ", timestamp=" + timestamp +
+               '}';
+    }
```

Comment:
This was kind of funny. When the test framework gets a failing test, it calls toString to print out the explanation. Since toString wasn't overridden here, it used the default Object.toString, which calls hashCode. But To's hashCode implementation throws an exception so that we don't accidentally use it in a hash collection. Needless to say, it took a minute to figure out what was happening, so I left the toString implemented.